src/ProviderRequestApprenticeTraining/SFA.DAS.ProviderRequestApprenticeTraining/Application/Commands/ExpireEmployerRequests/ExpireEmployerRequestsResponse.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+namespace SFA.DAS.ProviderRequestApprenticeTraining.Application.Commands.ExpireEmployerRequests
+{
+    public class ExpireEmployerRequestsResponse
```

Comment:
No information in the underlying response from API

Reply:
removed